code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -127,10 +129,10 @@ const styles = {
  */
 const ProgressDot = React.createClass({
   propTypes: {
-    levels: levelProgressShape,
+    level: levelProgressShape.isRequired,
```

Comment:
Yeah, I wish React would warn if you pass in extra props the component wasn't expecting.